modules/nw-control-dns-records-public-hosted-zone-infoblox.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-creating-dns-records-on-infoblox.adoc
+
+:_content-type: PROCEDURE
+[id="nw-control-dns-records-public-dns-zone-infoblox_{context}"]
+= Creating DNS records on an public DNS zone for Infoblox by using Red Hat External DNS Operator
+
+You can create DNS records on a public DNS zone for Infoblox by using the Red Hat External DNS Operator.
+
+.Procedure
+
+. Create a `secret` with Infoblox credentials:
```

Comment:
Per [ISG](https://www.ibm.com/docs/en/ibm-style?topic=elements-programming#keywords):

> Generally, specify the keyword type after the keyword. Keywords are not translated; so without the keyword type, users might be confused about what the keyword represents.

This guidance also applies to objects and custom resources in OpenShift. What type of thing is the `secret`?